lib/charms/mongodb/v0/config_server_interface.py

```diff
@@ -328,6 +364,22 @@ def update_keyfile(self, key_file_contents: str) -> bool:
 
         return True
 
+    def get_tls_statuses(self) -> Optional[StatusBase]:
+        """Returns statuses relevant to TLS."""
+        if self.is_mongos_tls_missing():
+            return BlockedStatus("mongos requires TLS to be enabled.")
+
+        if self.is_config_server_tls_missing():
+            return BlockedStatus("mongos has TLS enabled, but config-server does not.")
+
+        if not self.is_ca_compatible():
+            logger.error(
+                "mongos is integrated to a different CA than the config server. Please use the same CA for all cluster components."
+            )
+            return BlockedStatus("mongos CA and Config-Server CA don't match.")
+
+        return
```

Comment:
```suggestion
        return None
```

Reply:
ah this change must be made in the MongoDB charm, it looks like I forgot to save the file before making the change in my last PR to the MongoDB charm. I will do this there.

Reply:
https://github.com/canonical/mongodb-operator/pull/390/files